<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>0</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>140</y>
     <width>160</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>      蚁群算法</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ContinueButton">
      <property name="text">
       <string>下一轮</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AutoButton">
      <property name="text">
       <string>自动下一轮</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AntMinButton">
      <property name="text">
       <string>显示最短路径</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>360</y>
     <width>160</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>      遗传算法</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ContinueButton_2">
      <property name="text">
       <string>下一轮</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AutoButton_2">
      <property name="text">
       <string>自动下一轮</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="GenMinButton_2">
      <property name="text">
       <string>显示最短路径</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>280</y>
     <width>158</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>      最短长度</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="AntText">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>300</y>
     <width>158</width>
     <height>31</height>
    </rect>
   </property>
   <property name="tabStopWidth">
    <number>80</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="GeneticText">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>540</y>
     <width>158</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="GenText">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>610</y>
     <width>158</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>490</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>      最短长度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>530</y>
     <width>158</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>    基因型(32位)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>40</y>
     <width>158</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>创建地图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ResetButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>100</y>
     <width>158</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>重置地图</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>80</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>DP结果：</string>
   </property>
  </widget>
  <widget class="QLabel" name="DPLable">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>80</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CloseButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>746</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
